### ISSUE TITLE
Version should be 4.0.14 in package.json and missing v4.0.14 tag

### ISSUE THREAD

**Issue by tbouffard** [NaturalIntelligence/fast-xml-parser#539]
Currently, the latest version available on [npmjs](https://www.npmjs.com/package/fast-xml-parser) is `4.0.14` but
- the latest commit of the master branch still mentions 4.0.13 in `package.json`: https://github.com/NaturalIntelligence/fast-xml-parser/blob/45b01dbf8b56af2c8ee126be8c2e21ed79ae39f2/package.json
- there is no related in the repository

This makes the full changelog of the version hard to follow.
The only way to get the v4.0.14 details it is to access to this url: https://github.com/NaturalIntelligence/fast-xml-parser/compare/v4.0.13...master. But, the commits list will change in the future as far as new commits are pushed.

Here is the full changelog view:

![image](https://user-images.githubusercontent.com/27200110/214106471-f35e9106-6639-4a01-a248-31f8a52ac553.png)

**Would you like to work on this issue?**

- [ ] Yes
- [x] No, I could create a PR to update the version in the package(-lock).json files, but I cannot push a tag. 

### Notes

I didn't use the issue template for this description as it doesn't seem accurate for the report. Let me know if you want me to use it anyway.

**Comment by amitguptagwl** [comment]
I hope you should get it now

**Comment by tbouffard** [comment]
Thanks, that's good now. 😄
